src/components/data-exploration/generic-gene-table/CreateCellSetModal.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import {
+  Modal, Form, Button, Space, Select, InputNumber, Typography, Divider, Row, Alert,
+} from 'antd';// import PropTypes from 'prop-types';
+import PropTypes from 'prop-types';
+
+const { Title } = Typography;
+
+const CreateCellSetModal = (props) => {
+  const { selectedGenes, onCancel } = props;
+  const [form] = Form.useForm();
+  const intialFormValues = selectedGenes.map((gene) => ({ selectedGenes: gene }));
+
+  const createCellSet = (filterForm) => {
+    console.log('filter form is ', filterForm);
+  };
+
+  const comparisonOptions = [{
+    value: 'greaterThan',
+    label: 'Greater than',
+  }, {
+    value: 'lessThan',
+    label: 'Less than',
+  },
+  ];
+
+  const formHeight = selectedGenes.length < 5 ? selectedGenes.length * 75 : 300;
```

Comment:
why 75 per gene and limit the height to 300 (4 genes)? I see that there are still a lot of spaces that we can use. Aside from that, can you also reduce the spacing between each genes?

![image](https://user-images.githubusercontent.com/2862362/152348901-bcb38351-74fe-4f3c-919a-5ac24e6c80ab.png)

Reply:
I increased it because there are some long genes which dont fit in one line

Reply:
if we want all the rows to be aligned

Reply:
Other than that I wasnt sure how many genes should be shown before the scroll

Reply:
Try not to fix the `width`, so the gene names would fit on one line?

Reply:
Yeah this way the rows wont be aligned though?

Reply:
![image](https://user-images.githubusercontent.com/22479584/152357994-1adfe813-4b9b-4a7b-905d-6c1355226ade.png)

Reply:
My OCD didnt let me go with that :D

Reply:
@vickymorrison  what do you think about this?

Reply:
Maybe you can try something like 

    <Row>
      <Col flex="auto">Fill Rest</Col>
      <Col flex="100px">This is the comparison dropdown</Col>
      <Col flex="100px">This is the input field</Col>
    </Row>

Reply:
ill try it i always forget these `Row` and `Col` exist